github/pulls_reviews.go

```diff
@@ -0,0 +1,14 @@
+package github
+
+import "time"
+
+// No docs are out for this quite yet.
+type PullRequestReview struct {
```

Comment:
@shurcooL I considered using just `Review`. I did not go with it because these reviews are strictly for pull requests and the namespacing conventions I grokked here seemed to indicate that it would be most appropriate to be reasonably specific about what the review affects. Reviews are scoped to pull requests, therefore `PullRequestReview`. Does that seem reasonable?